### PR TITLE
Adds a selection of Exploration Items

### PR DESCRIPTION
**Pull request by EOBGames**
## About The Pull Request
Adds a selection of new items which are related to my (hopefully) WIP Exploration Department. The additions here will be necessary for further additions to the department, specifically in terms of mapping.

## Why It's Good For The Game
A lot of people have been complaining that gameplay is starting to get stale, and I've always thought space was underused. Hopefully this could get the ball rolling on something to remedy (at least partially) both of these problems.

## Changelog
:cl:
add: Added some clothing for the (WIP) Exploration Department
add: Added machinery for the Exploration Department
add: Added doors and tiles for the Exploration Department
add: A new firing pin that only works off the station Z-Level
imageadd: Added icons for the above new additions
code: Added defines for the department in economy.dm and research.dm
/:cl:

### CODE REVIEW

**Comment by Couls** [comment]
`file ./code/modules/clothing/under/jobs/exploration.dm is missing a trailing newline
file ./code/modules/research/machinery/departmental_protolathe.dm is missing a trailing newline
file ./code/game/objects/effects/decals/turfdecal/tilecoloring.dm is missing a trailing newline
file ./code/game/objects/items/circuitboards/machine_circuitboards.dm is missing a trailing newline`
add trailing newlines or face the wrath of travis for eternity

**Comment by kriskog** [comment]
> 
> 
> `file ./code/modules/clothing/under/jobs/exploration.dm is missing a trailing newline file ./code/modules/research/machinery/departmental_protolathe.dm is missing a trailing newline file ./code/game/objects/effects/decals/turfdecal/tilecoloring.dm is missing a trailing newline file ./code/game/objects/items/circuitboards/machine_circuitboards.dm is missing a trailing newline`
> add trailing newlines or face the wrath of travis for eternity

Add triple backticks on the line before and the line after or make badly readable error texts for eternity

**Comment by Couls** [comment]
I don't know what you're talking about 😏

**Comment by kriskog** [comment]
```
code\modules\vending\wardrobes.dm:429:error: /obj/item/clothing/under/rank/exploration/space_explorer: undefined type path
```

You should generally compile locally before you push, to weed out build errors :)

**Comment by EOBGames** [comment]
My local did work, but I wasn't sure if I should include the .dmb file, because I had a few other files that I didn't feel were ready to include with this PR that were included in the build. I can comment those changes out and rebuild, then submit the .dmb if that would help?

**Comment by kriskog** [comment]
Gotcha. No, the .dmb is ignored by git and we dont want that.
You should however make sure this branch compiles without the other changes, and ideally continue work on other stuff on a new branch which you could create from this one to copy the contents over.

**Comment by Couls** [comment]
you didn't checkmark some added files in the .dme

**Review by Couls — COMMENTED** [review]
Theres a whole file that isn't included in the tgstation.dme which is why your code won't compile, you need to checkmark it in the dme and then save it and push the saved .dme

File: code/modules/clothing/under/jobs/exploration.dm [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+/obj/item/clothing/under/rank/exploration
+	icon = 'icons/obj/clothing/under/exploration.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/under/exploration.dmi'
+
+/obj/item/clothing/under/rank/exploration/space_explorer
+	desc = "A hardwearing jumpsuit issued to members of the station's Exploration department, offering some protection from the dangers of space."
+	name = "explorer's jumpsuit"
+	icon_state = "exploration"
+	item_state = "o_suit"
+	armor = list("melee" = 5, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
```
Comment (Couls) [inline code/modules/clothing/under/jobs/exploration.dm]: This file

**Comment by HomocidlePancakes** [comment]
Mmmm, lavaland rework

**Comment by PepperPrepper** [comment]
So will the Exploration department focus on space an not lavaland?

**Comment by ArcaneMusic** [comment]
Even if the rest of this, for whatever reason, doesn't work, I WANT that explorer's firing pin.

**Comment by HomocidlePancakes** [comment]
When can we expect this being fully implemented?

**Comment by Tlaltecuhtli** [comment]
why not add spacepods if  you think space exploring is underused

**Comment by EOBGames** [comment]
> So will the Exploration department focus on space an not lavaland?

Current intention is to focus on space. Lavaland content could come after the department is in a working condition.

> When can we expect this being fully implemented?

The easy answer is that I don't know. The hard answer is that while the above is roughly 4 days worth of (very distracted) work, the next little while will be hectic for me IRL, and further changes will involve a deeper understanding of the code. Best estimate would be around 2-3 months at minimum, split into smaller PRs.

>why not add spacepods if you think space exploring is underused

I'm not quite sure what spacepods are, but one of the intended additions is a new shuttle to facilitate space exploration.

**Comment by davethwave** [comment]
Space pods are from goon https://wiki.ss13.co/Space_Pod

**Comment by EOBGames** [comment]
Well, the space pods seem like a good idea, but they don't quite fit the vision I've got for this. I'll keep them in mind though.

**Comment by EOBGames** [comment]
Yeah, based on the advice of ninjanomnom, I'm going to close this PR and come back later down the line when I have something more solid.

**Pull request closed without merging**